strings/strings.ko-KR.json

```
@@ -2399,23 +2399,23 @@
   "tech_genesis_effect": "우리 종족은 망한 것 같습니다. 그러니 다른 세계에 생명을 만들어서 정착시키는 계획을 세우세요.",
   "tech_star_dock": "우주 조선소",
   "tech_star_dock_effect": "고급 우주선을 제작할 수 있는 우주 조선시설 설계",
-  "tech_interstellar": "성간 구동계",
-  "tech_interstellar_effect": "실험용 엔진을 탑재하기 위해 특별히 설계된 우주선이 필요하지만, 수백 년이 아닌 수십 년 안에 인근의 다른 항성계에 도달할 수 있는 새로운 설계의 성간 구동계입니다.",
+  "tech_interstellar": "성간 구동기",
+  "tech_interstellar_effect": "실험용 엔진을 탑재하기 위해 특별히 설계된 우주선이 필요하지만, 수백 년이 아닌 수십 년 안에 인근의 다른 항성계에 도달할 수 있는 새로운 설계의 성간 구동기입니다.",
   "tech_genesis_ship": "제네시스 우주선",
   "tech_genesis_ship_effect": "먼 행성에 생명을 뿌리기 위한 우주선",
   "tech_generational_ship": "탈출 우주선",
   "tech_generational_effect": "다른 행성으로 탈출하기 위한 우주선",
   "tech_geck": "G.E.C.K.",
   "tech_geck_desc": "Global Ecosystem Construction Kit",
   "tech_geck_effect": "Design a terraforming device that could help improve a potential new homeworld.",
-  "tech_genetic_decay": "유전적 치료",
-  "tech_genetic_decay_effect": "유전자 파괴 완화",
-  "tech_stabilize_decay": "유전자 방어막",
-  "tech_stabilize_decay_effect": "유전적 치료 효과 증가",
+  "tech_genetic_decay": "유전자 치료",
+  "tech_genetic_decay_effect": "유전자 손상 문제를 조사하고 해결책을 찾아봅니다.",
+  "tech_stabilize_decay": "유전자 손상 안정화",
+  "tech_stabilize_decay_effect": "유전자 손상을 안정화하여 유전자 강화 효과를 크게 증폭시킵니다.",
   "tech_tachyon": "타키온 입자",
-  "tech_tachyon_effect": "타키온은 빛보다 빠르게 움직이는 입자입니다.",
-  "tech_warp_drive": "알쿠비에르 드라이브",
-  "tech_warp_drive_effect": "공간을 수축시키는 엔진 개발",
+  "tech_tachyon_effect": "타키온는 빛보다 빠르게 움직이는 입자로, 흥미진진한 발견입니다. 우주 여행에 혁명을 일으킬지도 모릅니다.",
```

여기 오타가 있습니다! '는'을 '은'으로 바꿔주세요.

```suggestion
  "tech_tachyon_effect": "타키온은 빛보다 빠르게 움직이는 입자로, 흥미진진한 발견입니다. 우주 여행에 혁명을 일으킬지도 모릅니다.",
```